cmd/minikube/cmd/config/util.go

```diff
@@ -138,7 +139,17 @@ func EnableOrDisableAddon(name string, val string) error {
 
 func enableOrDisableAddonInternal(addon *assets.Addon, cmd bootstrapper.CommandRunner, data interface{}, enable bool) error {
 	var err error
+	// check addon status before enable/disable
+	ok, err := addon.IsEnabled()
```

Comment:
Can you change this variable name to be little more informative?

Reply:
Address done. Thanks